vendor/vendor.json

```diff
@@ -675,11 +675,11 @@
 			"revisionTime": "2017-05-25T23:15:04Z"
 		},
 		{
-			"checksumSHA1": "ZkJRgexeNzNZzpw6YnedwoJl7pE=",
+			"checksumSHA1": "sdjkMflUxYtL/l7n7wYPS0Xj5c0=",
```

Comment:
Curious, why did vendor.json and the above two files have to change for this, they seem unrelated to the change being made?